plugins/outputs/influxdb/http.go

```diff
@@ -236,55 +238,66 @@ func (c *httpClient) CreateDatabase(ctx context.Context, database string) error
 	}
 }
 
+type dbrp struct {
+	Database        string
+	RetentionPolicy string
+}
```

Comment:
can you have the same database with different retention policies? The map below would seem to imply so

Reply:
Yes, this is the most common case actually.  I conceptualize this by thinking about all points being inside a database+retention_policy (not just the database).  If no rp is set, values will go into the default `autogen` rp.

This means you could have 2 exactly the same values at the same timestamp in the same database, and only by querying with the full "dbrp" will you be able to tell them apart.  It can be confusing to see because normally the retention policy is not seen.